docs/core/tools/dotnet-nuget-sign.md

```diff
@@ -0,0 +1,162 @@
+    [--hash-algorithm <HASHALGORITHM>]
+    [--timestamper <TIMESTAMPINGSERVER>]
+    [--timestamp-hash-algorithm <HASHALGORITHM>]
+    [--overwrite]
+    [-v|--verbosity <LEVEL>]
+
+dotnet nuget sign -h|--help
+```
+
+## Description
+
+The `dotnet nuget sign` command signs all the packages matching the first argument with a certificate. The certificate with the private key can be obtained from a file or from a certificate installed in a certificate store by providing a subject name or a SHA-1 fingerprint.
+
+## Arguments
+
+- **`package-path(s)`**
+
+  Specifies the file path to the package(s) to be signed. Multiple position arguments can be passed in to verify multiple packages.
+
+## Options
+
+- **`-o|--output`**
+
+  Specifies the directory where the signed package should be saved. If this option is not specified, by default the original package is overwritten by the signed package.
+
+- **`--certificate-path <PATH>`**
+
+  Specifies the file path to the certificate to be used in signing the package.
+
+> [!NOTE]
+> This command currently supports only `PKCS12 (PFX)` files that contain the certificate's private key.
+
+- **`--certificate-store-name <STORENAME>`**
+
+  Specifies the name of the X.509 certificate store to use to search for the certificate. Defaults to "My", the X.509 certificate store for personal certificates. This option should be used when specifying the certificate via --certificate-subject-name or --certificate-fingerprint options.
+
+- **`--certificate-store-location <STORELOCATION>`**
+
+  Specifies the name of the X.509 certificate store use to search for the certificate. Defaults to "CurrentUser", the X.509 certificate store used by the current user. This option should be used when specifying the certificate via --certificate-subject-name or --certificate-fingerprint options.
+
+- **`--certificate-subject-name <SUBJECTNAME>`**
+
+  Specifies the subject name of the certificate used to search a local certificate store for the certificate. The search is a case-insensitive string comparison using the supplied value, which will find all certificates with the subject name containing that string, regardless of other subject values. The certificate store can be specified by --certificate-store-name and --certificate-store-location options.
+
+> [!NOTE]
+> This command currently supports only a single matching certificate in the result. If there are multiple matching certificates in the result, or no matching certificate in the result, the command will fail.
+
+- **`--certificate-fingerprint <FINGERPRINT>`**
+
+   SHA-1 fingerprint of the certificate used to search a local certificate store for the certificate.
+
+- **`--certificate-password <PASSWORD>`**
+
+   Specifies the certificate password, if needed. If a certificate is password protected but no password is provided, the command will fail.
+
+> [!NOTE]
+> This command currently supports only noninteractive mode, so there won't be any prompt for a password at run time.
```

Comment:
```suggestion
  > [!NOTE]
  > This command currently supports only noninteractive mode, so there won't be any prompt for a password at run time.
```

Reply:
Same comment about "command" vs "switch"

Reply:
Thanks! This is going to say, there is no interactivemode for the sign command. So if the certificate is password protected, but no --certificate-password option is specified, there will be no prompt for a password at run time. (This is different from NuGet.exe command so I think it should be called out)
So changed into: `The sign command currently supports only noninteractive mode, so there won't be any prompt for a password at run time.`